src/app/shared/components/pricing-plans/pricing-plans.component.ts

```diff
@@ -51,6 +53,12 @@ export class PricingPlansComponent implements OnInit, OnDestroy {
 
   selectPlan(id) {
     this.store.dispatch(new MembershipUpdate({ id }));
+
+    // Add payment type for prime plan only
+    if (id === 1) {
+      this.store.dispatch( new UpdateSignupData({ payment_type: this.paymentMethod }));
```

Comment:
We are using the same `payment_type` parameter in `auth/upgrade/` as well. Currently its using `Stripe/Bitcoin` as payment types but it should be updated and use this value.See `stripeSignup` and `bitcoinSignup` methods in `users-billing-info.component.ts`.

Reply:
okay, @Waseem826 you are right.
@zainzafar90 can you please update there as well.